docs/src/app/components/master.jsx

```diff
@@ -1,16 +1,12 @@
 import React from 'react';
+import Title, { flushTitle } from 'react-title-component';
```

Comment:
Already on it - and wishing I'd linted the first working one before adding the rest. :sob: